projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+# Project Name
+_Include the name of project being proposed._
+
+VC API Implementation
+
+# Project Description
+_Include a description of the project, including what it does, why it is valuable, the origin and history, and other information that will allow others to understand what the project will include._
+
+The VC API is specification for a set of APIs for VC lifecycle management (https://w3c-ccg.github.io/vc-api/). This includes operations such as credential issuance, verification and exchange. It is a W3C CCG work item and, as of the submission of this proposal, it is in “draft community report” status. 
+
+The VC API’s design is informed by use cases across a range of domains. Several of these use cases are collected in a working group note (https://w3c-ccg.github.io/vc-api-use-cases/). 
+
+This project is an implementation of the VC API. The implementation aims to enable organizations and individuals to effortlessly conduct SSI operations over HTTP without requiring technical expertise, making it seamless to integrate into existing projects. 
+
+Easy to adapt new feature requirements and changes since the implementation is generic (can be used for different use-cases, energy use case is one of them). 
+
+The VC API specification was presented to the Open Wallet Foundation Architecture Special Interest Group during the 2023-04-24 meeting (https://github.com/openwallet-foundation/architecture-sig/wiki/2023-04-24-Meeting-Minutes). This project was used during the demonstration portion of the presentation.
```

Comment:
```suggestion
The VC API specification was presented to the [OpenWallet Foundation Architecture Special Interest Group during the 2023-04-24 meeting](https://github.com/openwallet-foundation/architecture-sig/wiki/2023-04-24-Meeting-Minutes). This project was used during the demonstration portion of the presentation.
```

Reply:
Thanks, fixed here: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/238bc1189325a9f3288eb6a661f9044dd5d06684

Reply:
Space removed in this commit: https://github.com/openwallet-foundation/project-proposals/pull/12/commits/e16e2999c7ec437df6fc47705fbb81106ea2af15